### PR TITLE
Add config schemas and Update readme

### PR DESCRIPTION
**Pull request by ashok133**
Added config.yaml schema 
Added hptune_config.yaml schema

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,59 +43,55 @@ storage-component.googleapis.com
 ```bash
 python3 -m venv venv
 source ./venv/bin/activate
-pip install ml-pipeline-gen
+pip3 install ml-pipeline-gen
```
Comment (TheMichaelHu) [inline README.md]: We're using a virtualenv so `pip` should be set correctly already, no need to specify `pip3`.

```sh
❯ source venv/bin/activate
❯ pip -V
pip 19.2.3 from /usr/local/google/home/humichael/Documents/forks/ml-pipeline-generator-python/venv/lib/python3.7/site-packages/pip (python 3.7)
❯ which pip
/usr/local/google/home/humichael/Documents/forks/ml-pipeline-generator-python/venv/bin/pip
❯ deactivate
❯ source venv2/bin/activate
❯ pip -V
pip 20.1.1 from /usr/local/google/home/humichael/Documents/forks/ml-pipeline-generator-python/venv2/local/lib/python2.7/site-packages/pip (python 2.7)
❯ which pip
/usr/local/google/home/humichael/Documents/forks/ml-pipeline-generator-python/venv2/bin/pip
```

File: README.md [hunk 2]
```diff
@@ -47,59 +43,55 @@ storage-component.googleapis.com
...
 ```
 
 Create a `config.yaml` by using the `config.yaml.example` template. See the
-[Input args](#input-args) section for details on the config parameters. Once the
+[schema](schema/CONFIG.md) for details on the config parameters. Once the
 config file is filled out, run the demo.
 
 ```bash
-python demo.py
+python3 demo.py
```
Comment (TheMichaelHu) [inline README.md]: See above comment, `python` should be set properly in the virtualenv, no need to specify `python2` vs `python3`.

File: schema/CONFIG.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+
```
Comment (TheMichaelHu) [inline schema/CONFIG.md]: nit: no need for this newline

File: schema/CONFIG.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+### config.yaml schema
+
+Below schema should be used when preparing a `config.yaml` file for models using the tool. Some parameters are optional and marked as such.
+
+<pre><code><b>project_id</b>: [project ID]
```
Comment (TheMichaelHu) [inline schema/CONFIG.md]: I get that you want to add emphasis to text in a code block and using html tags is the only way, just wondering if putting things in a code block is truly necessary.

If it is, nit: indent the stuff in the pre block so it looks like
```html
<pre>
    [stuff]
</pre>
```

Comment (TheMichaelHu) [reply schema/CONFIG.md]: Also, not sure what `<code>` does.

`<pre>stuff</pre>`
<pre>stuff</pre>

`<pre><code>stuff<code></pre>`
<pre><code>stuff<code></pre>

File: schema/HPTUNE_CONFIG.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (TheMichaelHu) [inline schema/HPTUNE_CONFIG.md]: same comments as config.md. Also, wondering if we should make a `docs/` directory to house these extra markdown files.

**Review by TheMichaelHu — CHANGES_REQUESTED** [review]
Thanks for cleaning up the docs again!

File: schema/HPTUNE_CONFIG.md (hunk 5 above)
Comment (ashok133) [reply schema/HPTUNE_CONFIG.md]: Ack. `docs/` makes more sense than `schema/`. Updated in new commit.

File: schema/CONFIG.md (hunk 4 above)
Comment (ashok133) [reply schema/CONFIG.md]: `<pre><code>...</pre></code>` should [allow](https://stackoverflow.com/a/4611735) for code that doesn't wrap around, but looking at GitHub's markdown rendering, it doesn't make any difference. Updated with `<pre>...</pre>`. 

Also, re: indenting, the `<pre>` tags honor spaces and tabs as is, so formatting code will result in extra indentation which will be rendered as spaces in markdown. Let me know if that's still an issue and I can look into other formats for the docs. Any suggestions?

File: schema/CONFIG.md (hunk 3 above)
Comment (ashok133) [reply schema/CONFIG.md]: Ack, removed.

File: README.md (hunk 2 above)
Comment (ashok133) [reply README.md]: That's nice. Didn't know venv enforces the python version with which it's created. Reverted all `python3` references except for when creating the venv.

File: schema/CONFIG.md (hunk 4 above)
Comment (TheMichaelHu) [reply schema/CONFIG.md]: ohh interesting. got it, nvm then

**Review by TheMichaelHu — APPROVED** [review]
LGTM 👍

**Pull request merged**